src/app/tournaments/[slug]/picks/page.tsx

```
@@ -676,12 +744,72 @@ export default function PicksPage({
 									{picks[match.id]?.predictedWinner && (
 										<div>
 											<label className="mb-2 block font-medium text-gray-700 text-sm">
-												Predicted Score
-											</label>
-											<div
-												className={`grid gap-4 ${tournament.format === "bo5" ? "grid-cols-3" : "grid-cols-2"}`}
-											>
-												{tournament.format === "bo3" ? (
+											{isFootball ? "Predicted Score" : "Predicted Score"}
+										</label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)